backend/donor-service/services/kafka.producer.js

```diff
@@ -0,0 +1,31 @@
+const config = require('../configs/config');
+const { Kafka, Partitioners } = require('kafkajs');
+
+// Connect and send the event
+async function produceEventToKafka(topic, telemetryEvent) {
+
+const kafka = new Kafka({
```

Comment:
I think this should initialised once only with try catch, same for producer as well, just producer.send should be called on request